docker/travis_test/entrypoint.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash                                                                                                                                                                                                        
```

Comment:
How about `#!/bin/sh -e` instead to not ignore errors?

Reply:
any reason to change shell?

why not just 
> set -e
?

Reply:
I think sticking with bash is fine, too (at least then we know what we get). I usually also use `set -e` separately. Shouldn't make any difference.